tools/cscs-scripts/build.sh

```diff
@@ -189,7 +189,7 @@ fi
 # pFUnit
 if [ ! -z ${PFUNIT_INSTALL_DIR+x} ]; then
     SERIALBOX_TESTING_FORTRAN=ON
-    SET_PFUNIT_PATH="-DpFUnit_DIR=${PFUNIT_INSTALL_DIR}"
```

Comment:
A question:
To pass PFUNIT to CMAKE I now need to pass its install-dir through "DCMAKE_PREFIX_PATH"?

E.g. here:
https://github.com/C2SM/spack-c2sm/blob/5a5339fce649c65d0ab2fd5904840331eb159440/packages/serialbox/package.py#L86

Reply:
Yes, either that or `-DPFUNIT_DIR=<path to the directory with PFUNIT.cmake file>` (or some other path, as described here https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure).